src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int max_num = 100;
```

Comment:
Let's use the style from the [guide](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) for constant naming

Reply:
And this